src/test/resources/feature/2-mittente/1_invioNotifiche/annullamentoNotifica.feature

```diff
@@ -0,0 +1,60 @@
+Feature: annullamento della notifica
+
+  @annullamentoNotifica
+  Scenario: PN-9242 - Mittente invia una notifica digitale al destinatario con indirizzo fornito dalla PA
+    Given PA - Si effettua la login tramite token exchange, e viene visualizzata la dashboard
+    And Nella pagina Piattaforma Notifiche cliccare sul bottone Invia una nuova notifica
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Informazioni preliminari
+    Then Nella section Informazioni preliminari si inseriscono i dati della notifica
+      | oggettoNotifica   | Pagamento rata IMU |
+      | descrizione       | PAGAMENTO RATA IMU |
+      | gruppo            | test-TA-FE-TEST    |
+      | codiceTassonomico | 123456A            |
+      | modalitaInvio     | A/R                |
+    And Cliccare su continua
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Destinatario
+    Then Nella section Destinatario si inseriscono i dati del destinatario
+      | soggettoGiuridico       | PF               |
+      | nomeCognomeDestinatario | Gaio Giulio      |
+      | codiceFiscale           | CSRGGL44L13H501E |
+    And Nella section Destinitario si clicca su "Aggiungi un indirizzo fisico" e si inseriscono i dati
+      | indirizzo | Via Roma |
+      | civico    | 20       |
+      | localita  | Milano   |
+      | comune    | Milano   |
+      | provincia | MI       |
+      | cap       | 20147    |
+      | stato     | Italia   |
```

Comment:
if create notification work, use here

Reply:
This test is duplicated, i don't know why is in this file. So i deleted this test